audio/vorbis-tools/Portfile

```
@@ -28,6 +28,8 @@ checksums           rmd160  f0f4f859b3a280dfe7ea9a13dc4c39ed3ed37c80 \
                     sha256  db7774ec2bf2c939b139452183669be84fda5774d6400fc57fde37f77624f0b0 \
                     size    1389947
 
+depends_build       port:pkgconfig
```

Since there is another port (pkgconf) that provides an implementation of the `pkg-config` program, a dependency on the pkgconfig port should always be written using `path:` syntax.
```suggestion
depends_build       path:bin/pkg-config:pkgconfig
```
(There are many ports that do not do this properly yet.)